### PR TITLE
Add rmspe on the V&V cases missing it

### PR DESCRIPTION
**Pull request by lin-yang-ly**
Add RMSPE calculation on ver-1a, ver-1b, ver-1d, ver-1fa, ver-1fb, ver-1g, and val-2b.

(Ref. #166)

### CODE REVIEW

**Review by simopier — CHANGES_REQUESTED** [review]
Great stuff!
Just some minor comments.

File: doc/content/verification_and_validation/val-2b.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The model applies 13.3 kPa of D$_2$ for 50 hours and 15 seconds followed by evac
 ## Results
 
 
-[val-2b_comparison] shows the comparison of the TMAP8 calculation and the experimental data. There is good agreement between the TMAP predictions and the experimental data with a root mean square percentage error of RMSPE = 19.41 %.
+[val-2b_comparison] shows the comparison of the TMAP8 calculation and the experimental data. There is reasonable agreement between the TMAP predictions and the experimental data with a root mean square percentage error of RMSPE = 19.41 %. Note that the agreement could be improved by adjusting the model parameters. It is also possible to perform this optimization with [MOOSE's stochastic tools module](modules/stochastic_tools/).
```
Comment (simopier) [inline doc/content/verification_and_validation/val-2b.md:44]: The link to the stochastic tools module does not work. Could you fix it?

**Review by simopier — CHANGES_REQUESTED** [review]
The link to the MOOSE STM does not work. Please update.

File: doc/content/verification_and_validation/val-2b.md (hunk 1 above)
Comment (lin-yang-ly) [reply doc/content/verification_and_validation/val-2b.md]: Done!

**Review by simopier — APPROVED** [review]
Thank you!

**Pull request merged**